apps/test/unit/code-studio/progressTest.js

```diff
@@ -34,3 +39,51 @@ describe('bestResultLevelId', function() {
     assert.strictEqual(bestResultLevelId([4, 5], progressData), 5);
   });
 });
+
+describe('initViewAs', function() {
+  let mockStore, mockSetViewType, mockQueryStringParse;
+  before(function() {
+    mockStore = {
+      dispatch: sinon.fake()
+    };
+
+    mockSetViewType = sinon.stub(viewAsRedux, 'setViewType');
+    mockQueryStringParse = sinon.stub(queryString, 'parse').returns({});
+  });
+
+  after(function() {
+    mockSetViewType.restore();
+    mockQueryStringParse.restore();
+  });
+
+  it('defaults to Student', function() {
+    initViewAs(mockStore, {});
+    assert(mockSetViewType.calledWith('Student'));
+  });
+
+  it('defaults to Teacher iff current user is a teacher', function() {
```

Comment:
nit: `iff` --> `if`, unless "if and only if" was intended?

Reply:
haha, it in fact was! Although I could remove it if it's confusing

Reply:
the "only if" part would be "does not default to Teacher if current user is not a teacher", so I guess I am confused because I don't see that part being tested here. Am I missing something?

Reply:
It's tested in the previous test. Together, they read:

- defaults to Student
- defaults to Teacher iff current user is a teacher

Reply:
close enough! :-)